### ISSUE TITLE
not able to install on Solus Linux

### ISSUE THREAD

**Issue by wuzhe** [nextdns/nextdns#69]
```
> sh -c 'sh -c "$(curl -sL https://nextdns.io/install)"'
ERROR: Unsupported OS: Linux
INFO: OS:
INFO: GOARCH: amd64
INFO: GOOS: linux
ERROR: Cannot detect running environement.
```

**Comment by PoorPocketsMcNewHold** [comment]
The installer doesn’t support Solus (Or the eopkg packages).

**Comment by eeeple** [comment]
Included since #146.
